testing/src/cypress/e2e/04-api-institution/03-instition-manage-designation.cy.ts

```diff
@@ -0,0 +1,115 @@
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization, {
+  ClientId,
+} from "../../custom-command/common/authorization";
+import CommonRegex from "../../custom-command/common/commonRegex";
+
+const institutionHelperActions = new InstitutionHelperActions();
+const commonRegexObject = new CommonRegex();
+
+const USERNAME = institutionHelperActions.getUserNameForApiTest();
+const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
+const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
+
+describe("[Designation details] ", () => {
```

Comment:
I am not sure if this was working in progress but I was under the assumption that we would be focusing on the UI tests and leaving the API ones for later. Please let me know if I am missing something 😉

Reply:
No you are right. But this test was already implemented by the time we were discussing to pause on it. So committed the changes.

Reply:
That's ok, resolving it 😉